app/dao/fact_billing_dao.py

```diff
@@ -826,23 +829,17 @@ def fetch_usage_for_organisation(organisation_id, year):
     letter_usages = _fetch_usage_for_organisation_letter(organisation_id, year_start, year_end)
     email_usages = _fetch_usage_for_organisation_email(organisation_id, year_start, year_end)
     for usage in sms_usages:
-        service_with_usage[str(usage.service_id)] = {
-            'service_id': usage.service_id,
-            'service_name': usage.service_name,
-            'free_sms_limit': usage.free_allowance,
+        # update sms fields
+        service_with_usage[str(usage.service_id)] |= {
```

Comment:
Nice use of |= and comment to go with it

Reply:
good to make use of some python 3.9 features!